src/config.cc

```diff
@@ -206,6 +206,11 @@ Config::Config() {
         false, new IntField(&RocksDB.max_bytes_for_level_multiplier, 10, 1, 100)},
       {"rocksdb.level_compaction_dynamic_level_bytes",
         false, new YesNoField(&RocksDB.level_compaction_dynamic_level_bytes, false)},
+      {"rocksdb.enable_db_paths", true, new YesNoField(&RocksDB.enable_db_paths, false)},
+      {"rocksdb.db_paths0", true, new StringField(&RocksDB.db_paths0, "")},
+      {"rocksdb.db_paths0_size_gb", true, new IntField(&RocksDB.db_paths0_size_gb, 0, 0, INT_MAX)},
```

Comment:
I prefer MiB, and you can just remove the `_gb` suffix, and describe it in the configuration file to users.